documentation/messaging_app_developer/master.adoc

```diff
@@ -0,0 +1,16 @@
+include::common/attributes.adoc[]
+
+= {Title}
+
+include::address-model.adoc[leveloffset=+1]
+
+include::address-space.adoc[leveloffset=+1]
+
```

Comment:
Do the address-model / address-space docs really belong here?  They are common concept type things that are probably most relevant to the tenant, but really likely to be info we present across a number of books.  A messaging app developer might choose to ignore the conceptual model part as they *only* care about connecting

Reply:
@rgodfrey, I'll remove them here and add them to the tenant master.adoc file instead. Also, since they are likely to be used in other guides as well in the future, I'll move those two files to the common folder to facilitate easier reuse.